docs/usage/getting-started/private-packages.md

```diff
@@ -360,8 +360,10 @@ The end-result looks like this:
 
 #### Yarn 2+
 
-Renovate doesn't support reading `npmRegistries` and `npmScopes` from `.yarnrc.yml`, so `hostRules` (or `npmToken`) and `npmrc` should be configured like above.
```

Comment:
Why have you removed this information?

Reply:
That's my fault! I suggested changes/edits to the line, as it was very hard to understand.

How about this then?

> #### Yarn 2+
>
> ##### `.yarnrc.yml` file limitations
>
> Renovate can _not_ read the following configuration items from `.yarnrc.yml` files:
> 
> - `npmRegistries`
> - `npmScopes`
>
> Use the following configuration options instead, to get a similar result:
> 
> - `hostRules` (or `npmToken`)
> - `npmrc`
>
> ...Rest of docs...

Reply:
Ah, yes, I think the text here is a little misleading still, it says "Renovate can not read [...] npmRegistries"
but that is the field that is written, so while it doesn't 'read' it, it's not totally unawares of it.

What about

```
Yarn 2+
.yarnrc.yml file limitations
Renovate can be configured to update the following items from .yarnrc.yml files:

npmRegistries
npmScopes
Use the following configuration options to update them:

hostRules (or npmToken)
npmrc
...Rest of docs...
```